modules/nested-elements/assets/js/editor/nested-repeater/views/widget-repeater.js

```diff
@@ -0,0 +1,97 @@
+const WidgetView = elementor.modules.elements.views.Widget;
+
+/**
+ * @extends {BaseElementView}
+ */
+export class WidgetRepeater extends elementor.modules.elements.views.BaseElement {
+	initialize() {
+		this.config = elementor.widgetsCache[ this.options.model.attributes.widgetType ];
+
+		this.once( 'container:created', this.onContainerCreatedOnce.bind( this ) );
+
+		WidgetView.prototype.initialize.apply( this, arguments );
+	}
+
+	events() {
+		const events = super.events();
+
+		events.click = ( e ) => {
+			// The aim is to skip only clicks on container elements.
+			const $parentsUntil = jQuery( e.target ).parentsUntil( '.e-container' );
+
+			if ( 4 === $parentsUntil.length || 1 === $parentsUntil.length ) {
```

Comment:
It's a guess, please use a stable solution

Reply:
Handled.